packages/api/lambdas/publish-granules.js

```diff
@@ -14,8 +14,14 @@ const handler = async (event) => {
 
   const promisedPublishEvents = event.Records.map(
     (record) => {
-      const granule = attr.unwrap(record.dynamodb.NewImage);
-      return publishSnsMessage(topicArn, granule);
+      const eventType = record.eventName;
+      const message = { event: eventType };
+      if (eventType === 'REMOVE') {
+        message.deletedAt = Date.now();
+      }
+      message.record = attr.unwrap(record.dynamodb.NewImage);
```

Comment:
Should this be `OldImage` in case of `REMOVE`, so it will have the granule information?  I want the `deletedAt` in the `record`: `message.record.deletedAt`

Reply:
Sure